Source/WTF/wtf/PlatformUse.h

```diff
@@ -313,7 +313,7 @@
 #if CPU(ARM64) && OS(DARWIN)
 // Only MacroAssemblerARM64 is known to build.
 // Building with TZONE_MALLOC currently disabled for all platforms.
-#define USE_TZONE_MALLOC 0
+#define USE_TZONE_MALLOC 1
```

Comment:
Be sure to turn this off before you land.

Note that EWS may fail to build because other dev are adding new `MAKE_ISO_ALLOCATED_*()` annotated types.